src/Fable.Transforms/FSharp2Fable.fs

```diff
@@ -148,11 +152,17 @@ let private getAttachedMemberInfo com ctx r nonMangledNameConflicts
                         if indexedProp then sign.Name, false, false
                         else Naming.removeGetSetPrefix sign.Name, isGetter, isSetter
                     // Setters can have same name as getters, assume there will always be a getter
-                    if not isSetter && nonMangledNameConflicts declaringEntityName name then
-                        sprintf "Member %s is duplicated, use Mangle attribute to prevent conflicts with interfaces" name
-                        // TODO: Temporarily emitting a warning, because this errors in old libraries,
-                        // like Fable.React.HookBindings
-                        |> addWarning com ctx.InlinePath r
+                    if not isSetter then
+                        if nonMangledNameConflicts declaringEntityName name then
+                            sprintf "Member %s is duplicated, use Mangle attribute to prevent conflicts with interfaces" name
+                            // TODO: Temporarily emitting a warning, because this errors in old libraries,
+                            // like Fable.React.HookBindings
+                            |> addWarning com ctx.InlinePath r
+
+                        if declaringEntityFields |> List.contains name then
+                            sprintf "Interface member %s.%s conflicts with existing member on type %s, add the Mangle attribute to the interface to prevent conflicts"
+                                sign.DeclaringType.TypeDefinition.FullName name declaringEntityName
+                            |> addError com ctx.InlinePath r
```

Comment:
As the one above is still a warning, maybe have this as a warning too until we change them both. We can also merge the two branches for simplicity, like:

```fsharp
if nonMangledNameConflicts declaringEntityName name
   || declaringEntityFields |> List.contains name then
        sprintf "Member %s is duplicated, add the Mangle attribute to the interface to prevent conflicts" name
        // TODO: Temporarily emitting a warning, because this errors in old libraries, like Fable.React.HookBindings
        |> addWarning com ctx.InlinePath r
```